params/extras/config.go

```diff
@@ -0,0 +1,352 @@
+// (c) 2024 Ava Labs, Inc. All rights reserved.
```

Comment:
```suggestion
// (c) 2025 Ava Labs, Inc. All rights reserved.
```

Reply:
I appreciate the comments on details that help us have a more consistent codebase, but since we are moving code between repositories, it is more important that they are textually in sync so the same diffs apply without merge conflicts.
This is a losing game to begin with, which is why we are pursuing the repository merger of coreth & subnet-evm.
Once we a single repos, making small improvements to code quality while moving it seems more reasonable.

Reply:
My apologies, totally right 💯